pkgs/applications/misc/stag/default.nix

```diff
@@ -11,14 +11,21 @@ stdenv.mkDerivation {
     sha256 = "1yrzjhcwrxrxq5jj695wvpgb0pz047m88yq5n5ymkcw5qr78fy1v";
   };
 
+  patches = [
+    # https://github.com/seenaburns/stag/pull/19
+    (fetchpatch {
```

Comment:
Please fetchpatch the individual commits instead. You can keep the PR number comment.

Reply:
done